app/src/renderer/vuex/modules/keybase.js

```diff
@@ -75,3 +69,35 @@ export default () => {
     mutations
   }
 }
+
+async function lookupId(state, keybaseId) {
```

Comment:
why do we ever need the ID?

Reply:
we get the ID from chain, the validators only enter the ID